generic/Tensor.c

```diff
@@ -2,7 +2,7 @@
 #define TH_GENERIC_FILE "generic/Tensor.c"
 #else
 
-static void torch_Tensor_(c_readTensorStorageSizeStride)(lua_State *L, int index, int allowNone, int allowTensor, int allowStorage, int allowStride,
```

Comment:
This PR won't be merged but
yes the only two places where I found this function to be called are the new() and set() function where all the arguments are true. I guessed they are legacy options that are not used anymore.